packages/server/api/src/app/ee/platform-billing/platform-billing.entity.ts

```diff
@@ -0,0 +1,62 @@
+import { PlatformBilling } from '@activepieces/ee-shared'
+import { Platform } from '@activepieces/shared'
+import { EntitySchema } from 'typeorm'
+import {
+    ApIdSchema,
+    BaseColumnSchemaPart,
+} from '../../database/database-common'
+
+export type PlatformBillingSchema = PlatformBilling & {
+    platform: Platform
+}
+
+export const PlatformBillingEntity = new EntitySchema<PlatformBillingSchema>({
+    name: 'platform_billing',
+    columns: {
+        ...BaseColumnSchemaPart,
+        platformId: ApIdSchema,
+        includedTasks: {
+            type: Number,
+        },
+        tasksLimit: {
+            type: Number,
+        },
+        includedAiCredits: {
+            type: Number,
+        },
+        aiCreditsLimit: {
```

Comment:
Let's make it nullable and consider null as unlimited, 0 is too implict for aiCreditsLimit and tasksLimit